src/merkle_proof.rs

```diff
@@ -173,6 +181,7 @@ impl FromBytes for PointerBlock {
     }
 }
 
+// FIXME not sure if this is used anywhere?
```

Comment:
We can delete `Trie` and `PointerBlock`, since we are importing `compute_state_hash` now.

Reply:
fixed